third_party/civetweb/civetweb.c

```diff
@@ -129,16 +129,26 @@ mg_static_assert(sizeof(void *) >= sizeof(int), "data type size check");
 
 #ifdef __MACH__
 
-#define CLOCK_MONOTONIC (1)
-#define CLOCK_REALTIME (2)
+#include <AvailabilityMacros.h>
```

Comment:
Thanks a lot for doing this (unrelated) work. Since this is a change in a third-party lib, it would be great if you could create a patch file (using e.g. git diff) and add it to the `third_party/patches/` folder and add a note in the `third_party/README` file, similar to the others. Thanks!